### ISSUE TITLE
ipfailover check port script does not work

### ISSUE THREAD

**Issue by kincl** [openshift/origin#19740]
While the keepalived process is starting up it throws this error:

```
Unable to access script `</dev/tcp/10.91.197.13/80`
Disabling track script chk_vip since not found
```

I think it is because /dev/tcp is a bash-specific redirection. In this example `10.91.197.13` is the IP address of the host that keepalived is running on

##### Version

Image: openshift/origin-keepalived-ipfailover:v3.9.0

##### Steps To Reproduce

Create a ipfailover deployment: `oc adm ipfailover vip`

##### Current Result

Check script fails to start

##### Expected Result

Check script runs successfully

##### Additional Information

none

**Comment by jwforres** [comment]
@openshift/sig-networking

**Comment by seferovic** [comment]
Hi, 

there is a fix for this error in #19890.

Why not use a shell script that does exactly this?
